config/config.go

```diff
@@ -530,12 +530,13 @@ func DefaultFuzzConnConfig() *FuzzConnConfig {
 
 // MempoolConfig defines the configuration options for the Tendermint mempool
 type MempoolConfig struct {
-	RootDir   string `mapstructure:"home"`
-	Recheck   bool   `mapstructure:"recheck"`
-	Broadcast bool   `mapstructure:"broadcast"`
-	WalPath   string `mapstructure:"wal_dir"`
-	Size      int    `mapstructure:"size"`
-	CacheSize int    `mapstructure:"cache_size"`
+	RootDir          string `mapstructure:"home"`
+	Recheck          bool   `mapstructure:"recheck"`
+	Broadcast        bool   `mapstructure:"broadcast"`
+	WalPath          string `mapstructure:"wal_dir"`
+	Size             int    `mapstructure:"size"`
```

Comment:
Should we plan to make these names more consistent? Eg. `MaxTotalTxs` and  `MaxTotalTxBytes` or something?